programs/airdrop/src/common.rs

```diff
@@ -15,6 +15,14 @@ pub struct State {
     pub vault: Pubkey,
     pub vault_bump: u8,
 
+    /// Seed and bump for this account. Should not be needed, but saved for
+    /// future use.
+    pub state_seed: [u8; 32],
+    pub state_bump: u8,
+
     /// Required signer when closing.
     pub close_authority: Pubkey,
+
+    /// Reserved for any future upgrades.
```

Comment:
Why is it better to add now vs when that upgrade actually happens?

Reply:
You dont want the struct to change size when in production. This is a common pattern that will make some migrations possible, although still very undesirable